diff --git a/PBStuff.rb b/PBStuff.rb
index e02e5d4..69716b9 100644
--- a/PBStuff.rb
+++ b/PBStuff.rb
@@ -602,6 +602,7 @@ POKEMONTOMEGASTONE = {
     :FLYGON => :FLYGONCREST,
     :ENTEI => :ENTEICREST,    :SUICUNE => :SUICREST,
     :RAIKOU => :RAICREST,   :CELEBI => :CELECREST,
+    :ARCHEOPS => :ARCHCREST, :ACCELGOR => :ACCELCREST,
   }
 
   SILVALLYCRESTABILITIES = {
diff --git a/Rejuv/Battle/Battle.rb b/Rejuv/Battle/Battle.rb
index ac10f16..757a518 100644
--- a/Rejuv/Battle/Battle.rb
+++ b/Rejuv/Battle/Battle.rb
@@ -11,6 +11,9 @@ class PokeBattle_Battle
     battler = @battlers[index]
 
     case battler.crested
+      when :ARCHEOPS
+        battler = @battlers[index]
+        battler.pbChangeStats(PBStats::ATTACK, -1, battler.pbOpposing1, :Intimidate)
       when :CASTFORM
         weathermoves = battler.moves.find { |move| [:SUNNYDAY, :RAINDANCE, :SANDSTORM, :HAIL, :SNOWSCAPE].include?(move.move) }
         return if weathermoves.nil?
@@ -73,6 +76,16 @@ class PokeBattle_Battle
       crestname = @battlers[index].fakeCrest ? "aura" : return
     end
     case battler.crested
+      when :ACCELGOR
+        newability = :SKILLLINK
+        pbShowAbilityBox(battler, attrname: getItemName(:ACCELCREST), crest: true)
+        pbDisplay(_INTL("{1} acquired {2}!", battler.pbThis, getAbilityName(newability)))
+        pbHideAbilityBox(battler)
+      when :ARCHEOPS
+        newability = :DEFIANT
+        pbShowAbilityBox(battler, attrname: getItemName(:ARCHCREST), crest: true)
+        pbDisplay(_INTL("{1} acquired {2}!", battler.pbThis, getAbilityName(newability)))
+        pbHideAbilityBox(battler)
       when :FURRET
         pbShowAbilityBox(battler, item: true)
         sublife = [(battler.totalhp / 4.0).floor, 1].max
diff --git a/Rejuv/Battle/Battler.rb b/Rejuv/Battle/Battler.rb
index 1306351..3e7295f 100644
--- a/Rejuv/Battle/Battler.rb
+++ b/Rejuv/Battle/Battler.rb
@@ -71,6 +71,11 @@ class PokeBattle_Battler
 
   def crestStats
     case @crested
+      when :ACCELGOR
+        @ability = :SKILLLINK
+        @spatk *= 1.2
+      when :ARCHEOPS
+        @ability = :DEFIANT
       when :SILVALLY
         @ability = PBStuff::SILVALLYCRESTABILITIES.keys.include?(@item) ? PBStuff::SILVALLYCRESTABILITIES[@item] : :SCRAPPY
         @attack *= 1.2
diff --git a/Rejuv/Definitions/itemtext.rb b/Rejuv/Definitions/itemtext.rb
index f3cb7a1..12eda3c 100644
--- a/Rejuv/Definitions/itemtext.rb
+++ b/Rejuv/Definitions/itemtext.rb
@@ -12367,4 +12367,21 @@ ITEMHASH = {
     :noUse => true,
   },
 
+  :ARCHCREST => {
+    :name => "Archeops Crest",
+    :desc => "Grants Competitive. Upon switching in, opponent Intimidates.",
+    :price => 0,
+    :crest => true,
+    :noUseInBattle => true,
+    :noUse => true,
+  },
+
+  :ACCELCREST => {
+    :name => "Accelgor Crest",
+    :desc => "Grants Skill Link. Increases Special Attack by 20%.",
+    :price => 0,
+    :crest => true,
+    :noUseInBattle => true,
+    :noUse => true,
+  },
 }
